articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,176 @@
+`Flutter` is an open-source UI software development kit that `Google` released in 2018. It uses Dart as its primary programming language. Flutter has led to the emergence of new software developers called `Flutter Developers`. 
+
+### Introduction
+
+Cross-platform applications enable companies to cut down on development costs. They eliminate the need to hire different software development teams. These applications also reduce the time and cost to manage different code bases. Flutter enables developers to create cross-platform applications. Due to these reasons, the need for Flutter Developers will continue to rise in the forseeable future. 
+
+
+### Table of Content
+- [Who is a Flutter Developer](#who-is-a-flutter-developer)
+- [Roles and Responsibilites of a Flutter Developer](#roles-and-responsibilites-of-a-flutter-developer)
+- [Flutter Developer Tools](#flutter-developer-tools)
+- [Flutter Developer Skills](#flutter-developer-skills)
+- [Mistakes to Avoid When Learning Flutter Development](#mistakes-to-avoid-when-learning-flutter-development)
+- [Getting a job as a Flutter Developer](#getting-a-job-as-a-flutter-developer)
+   - [Job Description Example](#job-description-example)
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who uses the Flutter framework to develop cross-platform applications. 
+
+### Roles and Responsibilities of a Flutter Developer
+
+1. **To Code Applications.**
+Flutter developers use the Flutter SDK to craft beautiful and interactive applications.
+
+2. **TO Test Applications.**
+Flutter developers test the functionality of the applications to make sure they do not crash. Testing ensures that the apps adhere to system and user requirements.
+
+3. **To Deliver Applications.**
+Flutter developers are involved in the deployment of applications. A mobile app is placed on `PlayStore` for Android applications or `AppleStore` for iOS applications. User feedback is then monitored and used to make changes to the applications.
+
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of changes they make on their projects. They also boost collaboration among software developers. As a result, they help increase overall productivity. 
+
+Examples of version control tools:
+
+[Git and Github](https://github.com/) - To use git and github you will need to install git to the code editor or IDE you are using and have an existing github account. All you need to do is write git commands to push your code to a github repository. From the repository other developers can help you improve your code. You can also set your repository as private.
```

Comment:
please rephrase this statement.
`To use git and github you will need to install git to the code editor or IDE you are using and have an existing github account.`
Do I need to install git to use github?

Reply:
@WanjaMIKE please explain how it's possible to use github without git on your local computer